packages/cf-component-icon/src/Icon.js

```diff
@@ -76,11 +288,18 @@ Icon.propTypes = {
     'widen',
     'wrench'
   ]).isRequired,
-  size: PropTypes.oneOf(['2x', '3x', '4x', 'large', 'xlarge']),
+  /**
+   * Sets a 1px border in color of passed in prop, defaults to Icon color
+   * default
+   */
   border: PropTypes.bool,
+  /**
+   * Renders icon with spinning animation
+   */
   spin: PropTypes.bool,
-  muted: PropTypes.bool,
-  white: PropTypes.bool,
+  /**
+   * TODO: What is this?
```

Comment:
I think it might be aria stuff. I cant see the file anymore so I don't know for sure.